test/functional/ui/cmdline_spec.lua

```diff
@@ -120,43 +120,43 @@ local function test_cmdline(linegrid)
         {1:~                                                 }|
         {1:~                                                 }|
         {1:~                                                 }|
-        {3:                                                  }|
+        {3:n                                                 }|
         {4:E121: Undefined variable: doesnotexist}            |
-        {4:E15: Invalid expression: doesnotexist}             |
-        {5:Press ENTER or type command to continue}^           |
       ]]}
       feed(':echoerr doesnotexist<cr>')
       screen:expect{grid=[[
-                                                          |
+        ^                                                  |
         {1:~                                                 }|
         {1:~                                                 }|
         {1:~                                                 }|
-        {3:                                                  }|
-        {4:E121: Undefined variable: doesnotexist}            |
-        {4:E15: Invalid expression: doesnotexist}             |
+        {1:~                                                 }|
+        {1:~                                                 }|
+        {1:~                                                 }|
+        {1:~                                                 }|
+        {3:n                                                 }|
         {4:E121: Undefined variable: doesnotexist}            |
-        {4:E15: Invalid expression: doesnotexist}             |
-        {5:Press ENTER or type command to continue}^           |
       ]]}
 
       feed(':echoerr doesnotexist<cr>')
       screen:expect{grid=[[
-                                                          |
+        ^                                                  |
         {1:~                                                 }|
-        {3:                                                  }|
-        {4:E121: Undefined variable: doesnotexist}            |
-        {4:E15: Invalid expression: doesnotexist}             |
-        {4:E121: Undefined variable: doesnotexist}            |
-        {4:E15: Invalid expression: doesnotexist}             |
-        {4:E121: Undefined variable: doesnotexist}            |
-        {4:E15: Invalid expression: doesnotexist}             |
-        {5:Press ENTER or type command to continue}^           |
+        {1:~                                                 }|
+        {1:~                                                 }|
+        {1:~                                                 }|
+        {1:~                                                 }|
+        {1:~                                                 }|
+        {1:~                                                 }|
+        {3:n                                                 }|
+        {4:E121: Undefined variable: doesnotexist}          |
       ]]}
```

Comment:
seems like this needs to be adjusted still.

```
[1m[31m[  ERROR   ][0m[0m [36m.../build/neovim/neovim/test/functional/ui/cmdline_spec.lua[0m @ [36m119[0m: [1mui/ext_cmdline redraws statusline on entering but not with scrolled messages[0m
test/functional/ui/screen.lua:531: Row 10 did not match.
Expected:
  |^                                                  |
  |{1:~                                                 }|
  |{1:~                                                 }|
  |{1:~                                                 }|
  |{1:~                                                 }|
  |{1:~                                                 }|
  |{1:~                                                 }|
  |{1:~                                                 }|
  |{3:n                                                 }|
  |*{4:E121: Undefined variable: doesnotexist}          |
Actual:
  |^                                                  |
  |{1:~                                                 }|
  |{1:~                                                 }|
  |{1:~                                                 }|
  |{1:~                                                 }|
  |{1:~                                                 }|
  |{1:~                                                 }|
  |{1:~                                                 }|
  |{3:n                                                 }|
  |*{4:E121: Undefined variable: doesnotexist}            |
```

Reply:
If I do that, the last line (`E121` error) disappears on my machine.

Reply:
@janlazo what do you mean? if you add 1 space to the last line it disappears?

Reply:
Yeah. Like this
`|*                                                                                   |`